koty.txt

```diff
@@ -0,0 +1 @@
+Lubie je�� koty i psy.
```

Comment:
w tym czymś  Marta-Be added some commits on 4 Feb, tam gdzie jest:

 @Marta-Be 	initial commit … 	865ef3a
	@Marta-Be 	lubie jesc koty … 	0fc45ae
	@Marta-Be 	Revert "lubie jesc koty" … 	a03ecdc
	@Marta-Be 	zdjecie 1 … 

moge sobie sprawdzić jak wygladal plik jpg, a jak klikne 3 kropeczki to pojawia sie opis (?) "w okazałości"